doc/source/news.rst

```diff
@@ -7,6 +7,79 @@
 News
 ====
 
+.. _release-8-0-2:
+
+Release 8.0.2 - 2018-04-29
+--------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`/reference/executables/grndb`][:ref:`grndb-force-truncate`] Improved
+  ``grndb recover --force-truncate`` option that it can be truncated even if
+  locks are left on the table.
+
+* [Windows] Use ``CONDITION_VARIABLE`` instead of Groonga's original implementation.
+
+* [:doc:`/reference/commands/logical_range_filter`] Added ``sort_keys`` option.
+
+* Added a new function ``time_format()``.
+  You can specify time format against a column of Time type.
+  You can specify with use format of ``strftime`` .
+
+* [:doc:`/reference/tokenizers`] Added the tokenizer that to be able to set options.
+  You can dynamically change tokenizer's behavior.
+  Specific options is as below.
+  * ``n`` : You can specify "N" of Ngram. For example, Ngram is trigram
+    when you can set "3" to "N".
+  * ``loose_symbol`` : The keywords including symbols are hit, even the
+    keywords not including symbols are hit also.
+  * ``remove_blank`` : Ignore blank.
+  * ``loose_blank`` : The keywords including blank are hit, even the
+    keywords not including blank are hit also.
+
```

Comment:
Could you add some example case for above options? For example, phone number or something else.